pkg/store/proxy.go

```diff
@@ -104,15 +123,69 @@ func (s *ProxyStore) Info(ctx context.Context, r *storepb.InfoRequest) (*storepb
 	res.MaxTime = MaxTime
 	res.MinTime = MinTime
 
-	for _, l := range s.selectorLabels {
```

Comment:
We need to remove selector flag as well then + CHANGELOG

Reply:
Actually didn't remove this. If specified label-sets are merged with the selector labels specified.